src/main/java/com/uber/jenkins/phabricator/conduit/Differential.java

```diff
@@ -119,7 +121,13 @@ public String getSummaryMessage(String phabricatorURL) {
 
     public String getPhabricatorLink(String phabricatorURL) {
```

Comment:
Meh, I guess it's just a semantic issue. Since all the callers need strings, it doesn't seem efficient or legible to keep converting to string when this is called. Maybe change the name of the function? I guess it's not a significant issue as long as it's documented.